oap-ape/ape-java/ape-server/src/main/java/com/intel/oap/ape/service/netty/server/NettyServer.java

```diff
@@ -0,0 +1,132 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.intel.oap.ape.service.netty.server;
+
+
+import java.net.InetAddress;
+import java.net.InetSocketAddress;
+
+import com.intel.ape.service.netty.NettyMessage;
+
+import io.netty.bootstrap.ServerBootstrap;
+import io.netty.channel.ChannelFuture;
+import io.netty.channel.ChannelHandler;
+import io.netty.channel.ChannelInitializer;
+import io.netty.channel.ChannelOption;
+import io.netty.channel.EventLoopGroup;
+import io.netty.channel.nio.NioEventLoopGroup;
+import io.netty.channel.socket.SocketChannel;
+import io.netty.channel.socket.nio.NioServerSocketChannel;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+/**
+ * A Netty server to handle requests from clients.
+ * This server will run standalone.
+ */
+public class NettyServer {

```

Comment:
do we need a factory to create `NettyServer` (like `ParquetDataRequestClientFactory`). I think we could reuse both `NettyServer` and `NettyClient` when we impl shuffle server/client.

Reply:
We don't need many `NettyServer`s, I think.

Reply:
`NettySever`s will be started from command line directly.